### PR TITLE
chore(docs): startup rsdoctor documents

### PR DESCRIPTION
**Pull request by easy1090**
## Summary
chore(docs): startup rsdoctor documents

img_v3_026p_3ae9627a-f4f3-45df-a0e4-92ed22b9127g
rjfdjLDzP3

## Checklist

- [ ] I have added changeset via `pnpm run change`.
- [x] I have updated the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: document/docs/en/index.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  actions:
+    - theme: brand
+      text: Introduction
+      link: /guide/start/intro
+    - theme: alt
+      text: Quick Start
+      link: /guide/start/quick-start
+  image:
+    src: https://lf3-static.bytednsdoc.com/obj/eden-cn/lognuvj/rsdoctor/logo/rsdoctor.png
+    alt: Rsbuild Logo
```
Comment (chenjiahan) [inline document/docs/en/index.md:17]:
```suggestion
    alt: Rsdoctor Logo
```

File: document/docs/public/rsdoctor-icon.png
Comment (chenjiahan) [inline document/docs/public/rsdoctor-icon.png:1]: 789KB is too large, the image should be minified.

File: document/docs/zh/guide/start/intro.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# 介绍
+
+## 什么是 Rsdoctor？
+
+**Rsoctor** 是一个面向**构建过程**与**构建产物**的**诊断与分析**的一站式工具，同时**支持多种前端类库**在项目中使用时的诊断与分析，来帮助开发者**快速发现**并**解决**问题。
```
Comment (chenjiahan) [inline document/docs/zh/guide/start/intro.mdx:5]: Too many bold text. Usually we only bold one or two keywords

File: document/docs/en/index.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    details: To display the compile execution time and process in the form of a timing diagram.
+    icon: 🚀
+  - title: Compile’s Actions
+    details: Visually view the compilation changes of each file in each loader.
+    icon: 🦄
+  - title: Bundle Analysis
+    details: Visually view the bundles & modules relationship, module reference relationship and repeated packages in detail.
+    icon: 🎯
+  - title: Bundle Diff
+    details: It is possible to monitor bundle degradation in CI.
```
Comment (chenjiahan) [inline document/docs/en/index.md:33]: “It is possible to” seems not necessary

File: document/docs/en/index.md (hunk 1 above)
Comment (easy1090) [reply document/docs/en/index.md]: fixed

File: document/docs/public/rsdoctor-icon.png
Comment (easy1090) [reply document/docs/public/rsdoctor-icon.png:1]: use cdn link.

File: document/rspress.config.ts [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  builderConfig: {
+    source: {
+      alias: {
+        '@components': path.join(__dirname, 'src/components'),
+        '@en': path.join(__dirname, 'docs/en'),
+        '@zh': path.join(__dirname, 'docs/zh'),
+      },
+    },
+    output: {
+      assetPrefix: process.env.OFFICAL_DOCS_PUBLIC_PATH || "",
```
[the remark this answers is not in the document]
Comment (easy1090) [reply document/rspress.config.ts]: It has been deleted, not used yet.

**Pull request merged**